frontend/public/components/monitoring.tsx

```diff
@@ -1273,55 +1273,75 @@ class SilenceForm_ extends React.Component<SilenceFormProps, SilenceFormState> {
-            sent out for alerts that match all the values or regular expressions.
+            Silences temporarily mute alerts based on a set of label selectors that you define.
+            Notifications will not be sent for alerts that match all the listed values or regular
+            expressions.
           </p>
-          <hr />
-          {Info && <Info />}
+        </div>
 
-          <div className="form-group">
-            <label className="co-required">Start</label>
-            <Datetime onChange={this.onFieldChange('startsAt')} value={data.startsAt} required />
-          </div>
-          <div className="form-group">
-            <label className="co-required">End</label>
-            <Datetime onChange={this.onFieldChange('endsAt')} value={data.endsAt} required />
-          </div>
-          <div className="co-form-section__separator" />
-
-          <div className="form-group">
-            <label className="co-required">Matchers (label selectors)</label>
-            <p className="co-help-text">
-              Alerts affected by this silence. Matching alerts must satisfy all of the specified
-              label constraints, though they may have additional labels as well.
-            </p>
-            <div className="row monitoring-grid-head text-secondary text-uppercase">
-              <div className="col-xs-5">Name</div>
-              <div className="col-xs-6">Value</div>
-            </div>
-            {_.map(data.matchers, (matcher, i) => (
-              <div className="row form-group" key={i}>
-                <div className="col-xs-10">
+        {Info && <Info />}
+
+        <div className="co-m-pane__body">
+          <form onSubmit={this.onSubmit}>
+            <div className="co-m-pane__body-group">
+              <div className="form-group row">
+                <div className="col-xs-9">
+                  <SectionHeading text="Duration" />
                   <div className="row">
-                    <div className="col-xs-6 pairs-list__name-field">
-                      <div className="form-group">
+                    <div className="col-xs-6">
+                      <label>Silence alert from...</label>
+                      <Datetime
+                        onChange={this.onFieldChange('startsAt')}
+                        value={data.startsAt}
+                        required
+                      />
+                    </div>
+                    <div className="col-xs-6">
+                      <label>Until...</label>
+                      <Datetime
+                        onChange={this.onFieldChange('endsAt')}
+                        value={data.endsAt}
+                        required
+                      />
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div className="co-m-pane__body-group">
+              <div className="form-group row">
+                <div className="col-xs-9">
```

Comment:
Any reason to not allow this to grow to the full 12 cols (`col-xs-12`) in order to take up less vertical space?

Reply:
Just following the new mocks for this page.

I don't have a preference, but if @cshinn feels it's better to make this text full width, I can change it (probably a follow-up PR?).